code/obj/item/clothing/uniforms.dm

```diff
@@ -1091,8 +1090,20 @@
 		icon_state = "suitR"
 		item_state = "suitR"
 
+	dress_navy
+		icon_state = "suitT-dress-alt"
+		item_state = "suitT-dress-alt"
+
+	navy
+		desc = "A navy suit and green necktie. An authoritative yet tacky ensemble."
+		icon_state = "suitT-alt"
+		item_state = "suitT-alt"
+
```

Comment:
Get rid of this. It was supposed to be deleted a while back when the clothing was taken out of this PR.